src/plugins/shell/shell.c

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file
+ *
+ * @brief Source for shell plugin
+ *
+ * @copyright BSD License (see doc/COPYING or http://www.libelektra.org)
+ *
+ */
+
+#include "shell.h"
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <errno.h>
+#include <kdberrors.h>
+#include <kdbhelper.h>
+
+
+int elektraShellOpen (Plugin * handle ELEKTRA_UNUSED, Key * errorKey ELEKTRA_UNUSED)
+{
+	// plugin initialization logic
+
+	return 1; // success
+}
+
+int elektraShellClose (Plugin * handle ELEKTRA_UNUSED, Key * errorKey ELEKTRA_UNUSED)
+{
+	// free all plugin resources and shut it down
+
+	return 1; // success
+}
+
+static int executeCommand(const char *cmdline)
+{
+	char *cmd[4];
+	cmd[0] = "/bin/sh";
+	cmd[1] = "-c";
+	cmd[2] = cmdline;
+	cmd[3] = NULL;
+	pid_t pid = fork();
+	if(pid == 0)
+	{
+		return execv("/bin/sh", cmd);
```

Comment:
I just pushed it because i had some problems building it and tried to figure out what was wrong. 
Actually, i was planning on redirecting stdout/stderr, but i can't think of any reason to use that now. 
Do you think parsing the output using regex or loading parts of it into elektra keys would be useful in any way ?